Package.swift

```diff
@@ -8,6 +8,8 @@ let package = Package(
     platforms: [.iOS(.v16), .macOS(.v13)],
     products: [
         .library(name: "Transformers", targets: ["Tokenizers", "Generation", "Models"]),
+        .library(name: "Hub", targets: ["Hub"]),
+        .library(name: "Tokenizers", targets: ["Tokenizers", "Models"]),
```

Comment:
```suggestion
        .library(name: "Tokenizers", targets: ["Tokenizers"]),
```

This would pull most of the library, wouldn't it?

Reply:
Yeah, I was getting an error with tokenizers when I didn't have the Models target, but I think that was just an issue with the test. I implemented the above change and fixed the test, and re-tested it in the external project.